plugins/node/opentelemetry-instrumentation-dns/README.md

```diff
@@ -0,0 +1,72 @@
+# OpenTelemetry DNS Instrumentation for Node.js
+
+[![Gitter chat][gitter-image]][gitter-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+This module provides automatic instrumentation for [`dns`](http://nodejs.org/dist/latest/docs/api/dns.html).
+
+For automatic instrumentation see the
+[@opentelemetry/node](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-node) package.
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/instrumentation-dns
+```
+
+## Usage
+
+```js
+const { NodeTracerProvider } = require('@opentelemetry/node');
+const { DnsInstrumentation } = require('@opentelemetry/instrumentation-dns');
+
+const provider = new NodeTracerProvider({
+  // be sure to disable old plugin
+  plugins: {
+    dns: { enabled: false, path: '@opentelemetry/plugin-dns' }
+  }
+});
+
+const dnsInstrumentation = new DnsInstrumentation({
+  // see under for available configuration
+});
+```
+
+### Zipkin
+
+If you use Zipkin, you must use `ignoreHostnames` in order to not trace those calls. If the server is local. You can set:
+
+```js
+const dnsInstrumentation = new DnsInstrumentation({
+  ignoreHostnames: ['localhost']
+});
+```
+
+### Dns Instrumentation Options
+
+Dns instrumentation has currently one option. You can set the following:
+
+| Options | Type | Description |
+| ------- | ---- | ----------- |
+| [`ignoreHostnames`](https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-instrumentation-dns/src/types.ts#L98) | `IgnoreMatcher[]` | Dns instrumentation will not trace all requests that match hostnames |
```

Comment:
most of the link in the readme point to the core repository, could you update them ?